src/routes/userRoutes.js

```diff
@@ -7,6 +7,7 @@ let app = express();
 let config = require('../config');
 let router = express.Router();
 
+// safer to get the secret from an env variable - Harold
```

Comment:
would be interested to learn how to do this... Which environment variable would we use?